packages/terra-core-docs/src/terra-dev-site/doc/section-header/example/LongTitleSectionHeader.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import SectionHeaderExampleTemplate from './SectionHeaderExampleTemplate';
 
-const LongTitleSectionHeader = () => <SectionHeaderExampleTemplate text="Long Title" exampleProps={{ text: 'This is a header with long text .This is an element provides a customizable header layout with a prominent title options for the elements.This component saves time and effort in designing a header from scratch and ensures consistency across a websites pages.' }} />;
+const LongTitleSectionHeader = () => <SectionHeaderExampleTemplate exampleProps={{ text: 'This is a header with long text .This is an element provides a customizable header layout with a prominent title options for the elements.This component saves time and effort in designing a header from scratch and ensures consistency across a websites pages.' }} />;
```

Comment:
This is not exactly related to the change, but as this text is user facing, it might be a good idea to fix typos such as no white space after a dot, extra white space before a dot, and "This is an element provides" needs rephrasing.